storage/ndb/rest-server/rest-api-server/internal/log/logc.go

```diff
@@ -40,7 +42,7 @@ func RegisterLogCallBack() {
 //export goLog
 func goLog(logMsg C.RS_LOG_MSG) {
 	level := log.Level(logMsg.level)
-	msg := C.GoString(&logMsg.message[0])
+	msg := fmt.Sprintf("RDRS native layer: %s", C.GoString(&logMsg.message[0]))
```

Comment:
Maybe optimisation Vincent did is applicable here as well?

Reply:
yes, i have fixed it